src/Text/Text.tsx

```diff
@@ -219,11 +220,12 @@ const Container = styled.p<IText>(
       letter-spacing: 0.75px;
     `}
 
-  margin: 0;
+    margin: 0;
     padding: 0;
-    color: ${theme.colors[color]};
     text-align: ${align};
     cursor: ${cursor};
     letter-spacing: 0.15px;
+    color: ${theme.colors[color]};
+    ${linkStyleOverride({ color: theme.colors[color] })}
```

Comment:
This work to override the colour, for the hover as well
It doesn't work for the active state and the font size, I don't think it's worth spending more effort into this as the `Text` component will be revamped in the future and we can find a more scalable way to handle it at this point